homeassistant/components/fibaro/sensor.py

```diff
@@ -115,16 +115,8 @@ async def async_setup_entry(
         # main sensors are created even if the entity type is not known
         entities.append(FibaroSensor(device, entity_description))
 
-    for platform in (
-        Platform.BINARY_SENSOR,
-        Platform.CLIMATE,
-        Platform.COVER,
-        Platform.LIGHT,
-        Platform.LOCK,
-        Platform.SENSOR,
-        Platform.SWITCH,
-    ):
-        for device in controller.fibaro_devices[platform]:
+    for platform_devices in controller.fibaro_devices.values():
+        for device in platform_devices:
```

Comment:
This change looks it doesn't need to be in the same PR and could be a separate PR

Reply:
This is absolutely correct, this change is not directly related. So I reverted this change to simplify the PR.